src/main/java/mat/client/measure/ManageMeasurePresenter.java

```diff
@@ -644,27 +643,24 @@ public void onFailure(Throwable caught) {
             public void onSuccess(FhirConvertResultResponse response) {
                 String outcome = response.getValidationStatus().getOutcome();
                 String errorReason = response.getValidationStatus().getErrorReason();
-                logger.log(Level.INFO, "Measure " + object.getId() + " conversion has completed. Outcome: " + outcome + " errorReason: " + errorReason);
+                logger.log(Level.WARNING, "Measure " + object.getId() + " conversion has completed. Outcome: " + outcome + " errorReason: " + errorReason);
                 setSearchingBusy(false);
-                if (!response.isSuccess()) {
-                    logger.log(Level.SEVERE, "Measure id " + object.getId() + " cannot be converted due to FHIR validation errors. Outcome " + outcome + " errorReason: " + errorReason);
-                    showFhirConversionError(MatContext.get().getMessageDelegate().getCannotConvertMeasureValidationFailedMessage());
-                } else {
-                    logger.log(Level.INFO, "Your measure was successfully converted to FHIR.");
-                }
-                //On conversion show the fhir measure validation errors.
-                showFhirValidationReport(response.getFhirMeasure().getId());
+                showFhirValidationReport(response.getFhirMeasure().getId(), true);
                 displaySearch();
             }
         });
 
     }
 
-    private void showFhirValidationReport(String measureId) {
+    private void showFhirValidationReport(String measureId, boolean converted) {
```

Comment:
If one is more common, converted or non-converted, would it make sense to add an overloaded method that defaults to the common argument.

Reply:
If it is only used twice, then ignore me.

Reply:
We could do that if there were more callers. We have just one method used in 2 places.

Reply:
Not this particular method, but I saw you had to update some tests to pass "false" in a few places. Having to maintain tests and runtime code may make having a default overloaded method worthwhile.

Reply:
It was a new test method to verify that the conversion throws an Exception when cqlSaved=false